UploadFile/PageEXT/Pag-Ext50416.BCPCustomerListExtension.al

```diff
@@ -0,0 +1,44 @@
+pageextension 50416 "BCP CustomerListExtension" extends "Customer List"
+{
+    layout
+    {
+        addfirst(factboxes)
+        {
+            part(BCPPicture; "BCP Upload")
+            {
+                ApplicationArea = All;
+                Caption = 'Upload File Factbox';
+                SubPageLink = "BCP Table ID" = const(18),
+                                "BCP No." = field("No.");
+            }
+        }
+    }
+
+    actions
+    {
+        addfirst(processing)
+        {
+            action("BCP UploadFile")
+            {
+                ApplicationArea = All;
+                Caption = 'Upload File';
+                ToolTip = 'Upload File to BC';
+                Promoted = true;
```

Comment:
Promoted property properties were outdated. For now you need to use ActionRef. You can hotfix it in VSCode by clicking on lamp with suggested fix on this line. Same for all actions

Reply:
Done